samples/snippets/src/main/java/com/example/storage/object/GetObjectContexts.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.storage.object;
```

Comment:
Each of these new samples will need to be moved to a separate PR, and the changes to the sample pom.xml need to be reverted.

The samples will then fail until the next release when the feature is available. The samples are run in three different dependency resolution scenarios:
1. direct dependency on `com.google.cloud:google-cloud-storage`
2. dependency on `com.google.cloud:google-cloud-storage` via `libraries-bom`
3. A build checking that the changes in the local branch don't break any previously working samples.